src/main/java/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package cinema.controller;
+
+import cinema.dto.UserRequestDto;
+import cinema.service.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto requestDto) {
```

Comment:
```suggestion
    public void register(@RequestBody UserRequestDto requestDto) {
```
method names should be verbs